index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <mate charset="utf-8">
```

Comment:
이부분은 오타같은데 한번 확인 부탁드릴게요!
```suggestion
    <mata charset="utf-8">
```